kerchunk/df.py

```diff
@@ -132,6 +139,18 @@ def refs_to_dataframe(
     **kwargs : dict
         Additional keyword arguments passed to fastparquet.
     """
+    # Taken from fsspec/implementations/reference.py
+    if isinstance(fo, str):
+        # JSON file
+        dic = dict(**(target_options or {}), protocol=target_protocol)
+        ref_fs, fo = fsspec.core.url_to_fs(fo, **dic)
+        with ref_fs.open(fo, "rb") as f:
```

Comment:
Unless we will need `ref_fs` again, this is a simpler version of the same
```suggestion
        with fsspec.open(fo, "rb", **dic) as f:
```